code/data.py

```diff
@@ -0,0 +1,80 @@
+'''
+Provide processed data ready for consumption by tensorflow model
+Return data always as numpy array
+'''
+
+import numpy as np
+import os
+from collections import Counter
+
+class NlpData:
```

Comment:
A single line of documentation which specifies inputs and outputs for the public methods would be useful.